internal/workflow/preprocessing.go

```diff
@@ -219,7 +219,11 @@ func (w *PreprocessingWorkflow) Execute(
 	}
 
 	if validateMetadata.Failures != nil {
-		result.validationError(ctx, ev, "metadata validation has failed", validateMetadata.Failures)
+		escapedFailures := make([]string, len(validateMetadata.Failures))
+		for _, f := range validateMetadata.Failures {
+			escapedFailures = append(escapedFailures, strings.ReplaceAll(f, identifySIP.SIP.Path, ""))
```

Comment:
We should remove the initial slash too, so it looks like a relative path. Also for consistency with the changes in https://github.com/artefactual-sdps/preprocessing-sfa/pull/71.